server/src/test/scala/org/http4s/armeria/server/ArmeriaServerBuilderSuite.scala

```diff
@@ -50,7 +50,9 @@ class ArmeriaServerBuilderSuite extends CatsEffectSuite with ServerFixture {
       IO(Thread.currentThread.getName).flatMap(Ok(_))
 
     case req @ POST -> Root / "echo" =>
-      Ok(req.body)
```

Comment:
An interesting issue occurred with the previous version of request body handling: the test suite hanged when running a test to check for exceeding the `maxRequestLength` limit. This was likely caused by an exception `com.linecorp.armeria.common.ContentTooLargeException` propagated from Armeria to fs2 Stream.  Ultimately, the request's body stream wasn't consumed resulting in not releasing some resources and hanging. It raises an interesting question about how we handle that exception from Armeria in the integration, but I'll leave it for another PR.